requirements.txt

```diff
@@ -0,0 +1,13 @@
+ansicolors==1.1.8
+boto3==1.5.32
+botocore==1.8.50
+docutils==0.16
+jmespath==0.9.3
+python-dateutil==2.8.1
+PyYAML==4.2b4
+s3transfer==0.1.13
+six==1.15.0
+elasticsearch[es1]==1.9.0
```

Comment:
I would rather not have the elasticsearch modules in the requirements.txt as that functionality is no longer maintained or recommended.

Reply:
Cool I can remove that.

Reply:
Removed in most recent commit